plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -348,7 +348,7 @@ func (p *pusher) createLogGroupAndStream() error {
 		_, err = p.Service.CreateLogGroup(&cloudwatchlogs.CreateLogGroupInput{
 			LogGroupName: &p.Group,
 		})
-		p.agentInfo.RecordOpData(time.Since(opStartTime), p.bufferredSize, err)
+		p.agentInfo.RecordOpData(time.Since(opStartTime), 1, err)
```

Comment:
Do we include the operation name as a dimension?

Are there separate metrics for errors from CreateLogGroup vs CreateLogStream vs PutLogEvents?

Reply:
That's a great question. Actually I'm of the mind that these `RecordOpData` calls should be removed for all but PutMetricData and PutLogEvents for now. I realized that these will muddle the API metrics for PutLogEvents since we're sending information about the LAST request sent, which could have been CreateLogGroup or other non-PutLogEvent APIs. I'll make this change real quick.